web/src/components/organisms/Asset/AssetList/hooks.ts

```diff
@@ -74,7 +59,7 @@ export default (projectId?: string) => {
     variables: {
       projectId: projectId ?? "",
       pagination: pagination(sort),
-      sort: toGQLEnum(sort?.type),
+      sort: sort?.type as GQLSortType,
```

Comment:
I think the original code, with the toGQLEnum is copied from reearth-web. I forget why we needed the toGQLEnum and all its complexity, but have you tested this thoroughly?

Reply:
I haven't fully implemented sort, but it works for this PR.